languages/elixir/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+In this exercise you're going to write a module to create and manipulate a map or high scores.
+
+You have 7 functions to implement, all related to returning and manipulating a map of high score data.
+
+### 1. Define a new high score map
+
+Define the `HighScore.new/0` function which doesn't take any parameters and returns new empty map of highscores.
+
+```elixir
+HighScore.new()
+# => %{}
+```
+
+### 2. Add players to the high score map
+
+Define `HighScore.add_player/3`, which is a function which takes 3 parameters:
```

Comment:
We usually phrase things such that there is also a description of what the function does in the task:

```suggestion
To add a player to the high score map, define `HighScore.add_player/3`, which is a function which takes 3 parameters:
```

Or something similar. This comment also applies to the other tasks.